test/intl402/NumberFormat/prototype/formatRange/x-greater-than-y-throws.js

```diff
@@ -0,0 +1,20 @@
+// Copyright (C) 2021 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-Intl.NumberFormat.prototype.formatRange
+description: >
+  "formatRange" basic tests when argument  x > y, BigInt included, throw a RangeError exception.
+info: |
+  1.1.21 PartitionNumberRangePattern( numberFormat, x, y )
+  (...)
+  1.1.21_2.a. If y is a mathematical value and y < x, throw a RangeError exception.
+features: [Intl.NumberFormat-formatRange]
+---*/
+
+const nf = new Intl.NumberFormat("en-US", {signDisplay: "exceptZero"});
+
+// If x > y, throw a RangeError exception.
```

Comment:
Here and below in the formatRangeToParts section: this test should be more thorough.  It should cover all the cases from PartitionNumberRangePattern items 2-4, including all sub-bullets.  This stuff:

```
If x is a mathematical value, then
  If y is a mathematical value and y < x, throw a RangeError exception.
  Else if y is -∞, throw a RangeError exception.
  Else if y is -0 and x ≥ 0, throw a RangeError exception.
Else if x is +∞, then
  If y is a mathematical value, throw a RangeError exception.
  Else if y is -∞, throw a RangeError exception.
  Else if y is -0, throw a RangeError exception.
Else if x is -0, then
  If y is a mathematical value and y < 0, throw a RangeError exception.
  Else if y is -∞, throw a RangeError exception.
```

Reply:
Coverage was updated to cover all the bullets :)